samples/basic/shell/src/AuthenticatedLayout.tsx

```diff
@@ -40,17 +40,30 @@ const renderSection: RenderSectionFunction = (elements, index, level) => {
     );
 };
 
-export default function AuthenticatedLayout() {
+export interface AuthenticatedLayoutProps {
+    sessionManager: SessionManager;
+}
+
+export function AuthenticatedLayout({ sessionManager }: AuthenticatedLayoutProps) {
     const session = useSession() as Session;
 
+    const navigate = useNavigate();
+
     const handleModulesMessage = useCallback((data: unknown) => {
         console.log("[sample] Message received from a module: ", data);
     }, []);
 
     useApplicationEventBusListener("write-to-host", handleModulesMessage);
 
-    const navigationItems = useNavigationItems();
+    const onDisconnect = useCallback(async (event: MouseEvent<HTMLButtonElement>) => {
+        event.preventDefault();
```

Comment:
There is no await in this method, is the async still necessary?

Reply:
Fixed!